app/models/pawn.rb

```diff
@@ -1,4 +1,24 @@
 class Pawn < Piece
+
+	#en passant status of self
+	def update_en_passant_status(end_x, end_y)
+		if (self.pos_y - end_y).abs == 2
+			self.update_attributes(en_passant: true)
+		else
+			self.update_attributes(en_passant: false)
+		end
+	end
+
+	#determine if pawn can capture opposing pawn en passant
+	def can_capture_en_passant(opponent_pawn)
+		if opponent_pawn.updated_at > self.game.updated_at && opponent_pawn.en_passant == true 
+			return true
+		else
+			return false
+		end
+	end
+
+
```

Comment:
I don't know if this logic is completely sound, but it does pass my test finally (and fail if I assert the opposite so I know it's doing something). Might be good enough for now to merge in?